examples/gradle-kt-dsl/lib/build.gradle.kts

```diff
@@ -0,0 +1,33 @@
+val accpVersion = "1.6.1"
+val archEnv = System.getenv("ARCH")?: "x86_64"
+//val fips = if (System.getenv("FIPS") == null) "" else "-FIPS"
```

Comment:
should we also pull down + install the FIPS artifact and assert its FIPSness?

Reply:
This line is commented out because I'm not sure if we will be publishing the FIPS artifacts since AWS-LC does not have its FIPS certificate yet.